### ISSUE TITLE
Wrong variable in parameter for phpDoc IN DataBuilder

### ISSUE THREAD

**Issue by kunj1988** [magento/magento2#19974]
### Summary (*)
1. You can find it in any Magento 2.1.x, 2.2.x AND 2.3
2. File path `vendor/magento/module-catalog/Model/Layer/Filter/Item/DataBuilder.php` OR `app/code/Magento/Catalog/Model/Layer/Filter/Item/DataBuilder.php`.

### Examples (*)
```
/**
     * Add Item Data
     *
     * @param string $label
     * @param string $value
     * @param int $count
     * @return void
     */
    public function addItemData($label, $value, $count)
    {
        $this->_itemsData[] = [
            'label' => $label,
            'value' => $value,
            'count' => $count,
        ];
    }
```

### Proposed solution
```
 /**
     * Add Item Data
     *
     * @param string $label
     * @param string $label
     * @param int $count
     * @return void
     */
    public function addItemData($label, $value, $count)
    {
        $this->_itemsData[] = [
            'label' => $label,
            'value' => $value,
            'count' => $count,
        ];
    }
```

**Comment by magento-engcom-team** [comment]
Hi @kunj1988. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@kunj1988 do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [x] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [x] 6. Add label `Issue: Confirmed` once verification is complete. 

- [x] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
@engcom-backlog-nazar Thank you for verifying the issue.

Unfortunately, not enough information was provided to created internal ticket. Please consider adding the following:
- [x] Add `"Component: "` label(s) to this ticket based on verification result. If uncertain, you may follow the best guess

Once all required information is added, please add label `"Issue: Confirmed"` again.
Thanks!

**Comment by magento-engcom-team** [comment]
@engcom-backlog-nazar Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-97334`, `MAGETWO-97335` were created

**Comment by orlangur** [comment]
@kunj1988 such changes should be proposed directly as pull requests ;)

**Comment by magento-engcom-team** [comment]
Hi @kunj1988. Thank you for your report. 
The issue has been fixed in magento/magento2#19992 by @kunj1988 in 2.3-develop branch
Related commit(s):
 - [4ed79dc73482011e1a9023c9e33a7866b56a05c9](https://github.com/magento/magento2/commit/4ed79dc73482011e1a9023c9e33a7866b56a05c9)
 - [82e51caf89389c9ffbfb1d85b33421a65b491012](https://github.com/magento/magento2/commit/82e51caf89389c9ffbfb1d85b33421a65b491012)
 - [ee9a2be43f9a291546dd4ab373002ae42ade431b](https://github.com/magento/magento2/commit/ee9a2be43f9a291546dd4ab373002ae42ade431b)

The fix will be available with the upcoming 2.3.1 release.
